pages/linux/iwconfig.md

```diff
@@ -0,0 +1,20 @@
+# iwconfig
+
+> Configure and show the parameters of a wireless network interface.
+> More information: <https://linux.die.net/man/8/iwconfig>.
+
+- Show the parameters and statistics of all the interfaces:
+
+`iwconfig`
+
+- Show the parameters and statistics of the specified interface:
+
+`iwconfig {{interface}}`
+
+- Set the ESSID (network name) of the specified interface (e.g., eth0 or wlp2s0):
```

Comment:
You can just mention the interface in the example itself, rather than mentioning `(e.g., eth0 or wlp2s0)`. After all, that is the purpose of the example. To give an example :)